crates/swc/tests/tsc-references/templateStringWithEmbeddedNewOperator.2.minified.js

```diff
@@ -1,2 +1,2 @@
 //// [templateStringWithEmbeddedNewOperator.ts]
-"abc".concat(new String("Hi"), "def");
+new String("Hi");
```

Comment:
Probably isn't important with `String`, but it's common for `Map`/`WeakMap` to be used. Are we eliminating built in constructors?

Reply:
Yes, but we are not dropping `String`. I think we should, though.

https://github.com/swc-project/swc/blob/7d5b5444580ab334d822bad2f78a741760f0dd61/crates/swc_ecma_minifier/src/compress/pure/misc.rs#L1096-L1100